src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -128,6 +136,17 @@ export const TradeConfirm = ({ history }: RouterProps) => {
     buyTxid,
   ])
 
+  useEffect(() => {
+    ;(async () => {
+      const buyAssetId = trade?.buyAsset.assetId
+      const sellAssetId = trade?.sellAsset.assetId
+      if (!buyAssetId || !sellAssetId) return ''
+      const swapperManager = await getSwapperManager(flags)
+      const bestSwapper = await swapperManager.getBestSwapper({ buyAssetId, sellAssetId })
+      setSwapperName(bestSwapper?.name ?? '')
+    })()
+  }, [flags, trade])
```

Comment:
Wondering if we want to be extra safu here and have 0 deps:
- changing flags would re-render swapper
- `trade` should always be defined, and changing it with user input means we unmounted `<TradeConfirm />` and re-mounted it

Looks like there's no case where we legitimately *need* the deps, but there's room for bugs by having them